src/Descriptor.cpp

```diff
@@ -36,7 +37,7 @@ const char *short_name_of(DescriptorState state) {
     return "<UNK>";
 }
 
-Descriptor::Descriptor(uint32_t descriptor) : channel_(descriptor), login_time_(ctime(&current_time)) {}
+Descriptor::Descriptor(uint32_t descriptor) : channel_(descriptor), login_time_(current_time) {}
```

Comment:
I hunted around to find how it was gettin current_time, I see it's a global right now ;)

Reply:
Sadly so; yes. I think it might serve us well to start calling out globals more obviously. It's not clear here. Maybe a naming convention for now/